src/validator.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const validateGuess = guessNumber => {
```

Comment:
you should also check if all digits are different
![image](https://github.com/mate-academy/node_bulls-and-cows/assets/147749101/34fcaacf-902e-435f-88ef-b37d95f7c3b1)


![image](https://github.com/mate-academy/node_bulls-and-cows/assets/147749101/52e4461d-823e-4885-9810-857e59d19c7b)